rdr_service/repository/questionnaire_response_repository.py

```diff
@@ -198,7 +203,12 @@ def get_interest_in_sharing_ehr_ranges(cls, participant_id, session: Session):
                 consent_answer = response.get_single_answer_for(code_constants.EHR_CONSENT_QUESTION_CODE)
                 if consent_answer:
                     # ignore any EHR responses that are not validated
-                    if response.id not in validated_ehr_id_list and not skip_validation_check:
+                    # Note: only check for validated EHR if the consent was authored after the response->consent data
```

Comment:
@kenny-skaggs FYI, and fixable with a `consent_response` backfill? I found 491 EHRConsentPII responses authored since this date that do not have a corresponding `consent_response` table record.  (Was not filtering on test pids or duplicate `questionnaire_response_id` classifications).